front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -242,29 +285,35 @@ const ScenarioV2 = () => {
                     <div className="scenario-details-description">{scenario.description}</div>
                   </div>
                 )}
-                {displayTrainScheduleManagement !== MANAGE_TRAIN_SCHEDULE_TYPES.none && infra && (
-                  <TimetableManageTrainScheduleV2
-                    displayTrainScheduleManagement={displayTrainScheduleManagement}
-                    setDisplayTrainScheduleManagement={setDisplayTrainScheduleManagement}
-                    setTrainResultsToFetch={setTrainResultsToFetch}
-                    trainIdToEdit={trainIdToEdit}
-                    setTrainIdToEdit={setTrainIdToEdit}
-                    infraState={infra.state}
-                  />
-                )}
-                {infra && (
-                  <TimetableV2
-                    setDisplayTrainScheduleManagement={setDisplayTrainScheduleManagement}
-                    trainsWithDetails={trainsWithDetails}
-                    infraState={infra.state}
-                    trainIds={timetable.train_ids}
-                    selectedTrainId={selectedTrainId}
-                    conflicts={conflicts}
-                    setTrainResultsToFetch={setTrainResultsToFetch}
-                    setSpaceTimeData={setTrainSpaceTimeData}
-                    setTrainIdToEdit={setTrainIdToEdit}
-                    trainIdToEdit={trainIdToEdit}
-                  />
+                {microMacroSwitch}
```

Comment:
No need to use a const, you can remove it and directly use the component :)
```suggestion
                <MicroMacroSwitch isMacro={isMacro} onClickButton={onClickMicroMacroButton} t={t} />
```

Reply:
Done